Makefile

```diff
@@ -19,10 +19,10 @@ default:
 		$(MAKE) deps
 		$(MAKE) all
 deps:
-		bash -c "godep restore"
+		bash -c "./scripts/deps.sh"
 test:
 		bash -c "./scripts/test.sh $(TEST)"
```

Comment:
To improve the UX you could consider adding 

```
test-unit:
    bash -c "./scripts/test.sh unit"
test-integration:
    bash -c "./scripts/test.sh integration"
```

Otherwise users will need to set the env TEST to a valid value (unit or integration)